Add other intro pages (#5)
brigadecore/docs-portuguese: conteudo/introducao/proximospassos.md

---
title: Próximos Passos

> description: Próximos passos e dicas para aprender Brigade

Reviewer: ```suggestion
description: Próximos passos e dicas para aprender sobre o Brigade
```

> title: Próximos Passos
> description: Próximos passos e dicas para aprender Brigade
> section: introducao
> weight: 4

Reviewer: ```suggestion
título: Próximos Passos
descrição: Próximos passos e dicas para aprender Brigade
seção: introdução
peso: 4
```

Author: fixed. i am keeping titles in English.

aliases:
  - /proximospassos.md
  - /introducao/proximospassos.md
---

> Agora que você leu todo o material introdutório, esperamos que você esteja estusiasmado com o Brigade da mesma forma que nos estamos.

Reviewer: ```suggestion
Agora que você leu todo o material introdutório, esperamos que você esteja estusiasmado com o Brigade da mesma forma que nós estamos.
```

Author: fixed

> Doravante, a documentção será mais contextual e detalhada. Na secção [Advanced Topics], nos iremos 

Reviewer: ```suggestion
Doravante, a documentção será mais contextual e detalhada. Na seção [Advanced Topics], nós iremos 
```

Author: fixed

> Doravante, a documentação será mais contextual e detalhada. Na seção [Tópicos Avançados], nós iremos 

Reviewer: ```suggestion
A partir de agora, a documentação será mais contextual e detalhada. Na seção [Tópicos Avançados], nós iremos 
```

> detalhar a [architecture and design] do Brigade, explorar as diferentes [roles of interaction] existentes,

Reviewer: ```suggestion
detalhar a [arquitetura e design] do Brigade, explorar as diferentes [roles of interaction] existentes,
```

> mergulhar na parte de [scripting guides], discutir sobre o [desenvolvimento do próprio Brigade] e muito mais.

Reviewer: ```suggestion
mergulhar na parte de [guias de scripting], discutir sobre o [desenvolvimento do próprio Brigade] e muito mais.
```

> A medida que você navegar pela documentação, se você perceber alguma lacuna or área necessitando nossa atenção, por favor crie um [preencha um issue].

Reviewer: ```suggestion
A medida que você navegar pela documentação, se você perceber alguma lacuna ou área necessitando nossa atenção, por favor crie um [preencha um issue].
```

Aproveite a Leitura!

[Tópicos Avançados]: ../topics/index.md
[arquitetura e design]: ../topics/design.md
[roles de interação]: ../topics/roles.md

> [scripting guides]: ../topics/scripting/index.md

Reviewer: ```suggestion
[guias de scripting]: ../topics/scripting/index.md
```
